src/Bridges/bridge.jl

```diff
@@ -30,6 +30,20 @@ type `F`-in`S` created by the bride `b` in the model (i.e., of length equal to t
 """
 MOI.get(b::AbstractBridge, ::MOI.ListOfConstraintIndices{F, S}) where {F, S} = CI{F, S}[]
 
+"""
+    supportedconstrainttypes(BT::Type{<:AbstractBridge})::Bool
+
+Return a list of the types of constraints that can be bridged with a bridge of type `BT`.
+"""
+function supportedconstrainttypes end
```

Comment:
Is this evidence for why I think this is necessary for all optimizers?

Reply:
Or the opposite, an evidence that `supportedconstrainttypes` is a bad idea since it does not support bridges that can bridge infinitely many constraints (e.g. bridges that bridge AnyFunction-in-SomeSet)